conan_support.cmake

```diff
@@ -163,10 +166,78 @@ function(conan_install)
 endfunction()
 
 
+function(conan_version_parse result conan_version conan_version_raw)
+    set(${result} FALSE PARENT_SCOPE)
+
+    if(NOT conan_version_raw)
+        message(FATAL_ERROR "CMake-Conan: conan_version_parse requires three parameters")
+    endif()
+
+    string(REGEX MATCH "[0-9]+\\.[0-9]+\\.[0-9]+" _conan_version ${conan_version_raw})
+    if(NOT _conan_version)
+        return()
+    endif()
+    string(REPLACE "." ";" conan_version_list "${_conan_version}")
+    list(LENGTH conan_version_list conan_version_list_length)
+    if(conan_version_list_length EQUAL 3)
+        set(${result} TRUE PARENT_SCOPE)
+    endif()
+    set(${conan_version} ${_conan_version} PARENT_SCOPE)
+endfunction()
+
+
+function(conan_get_current_version conan_command conan_current_version)
```

Comment:
``conan_get_version`` sounds shorter but still very meaningful, maybe the ``current`` can be dropped

Reply:
Makes sense, renamed!